core/dbt/graph/selector_spec.py

```diff
@@ -80,7 +80,7 @@ def __post_init__(self):
     def default_method(cls, value: str) -> MethodName:
         if _probably_path(value):
             return MethodName.Path
-        elif value.lower().endswith(".sql"):
+        elif value.lower().endswith((".sql",".py")):
```

Comment:
(Out of scope for this PR, but: should we also have `".csv"` here?)

Reply:
is this for `dbt seed`?

Reply:
yup

Reply:
I'll defer to @jtcohen6 and @ChenyuLInx on this decision. I'd have to review the dependencies / code in more detail to understand whether including `.csv` would require additional changes, and I'm sure you have a better sense for that. If it's simply a matter of adding the `.csv` to this one line to slightly expand this PR, I definitely don't object! ty

Reply:
#5578 for this comment